### ISSUE TITLE
Duplicated Keys

### ISSUE THREAD

**Issue by david-morris** [aboveyunhai/chakra-dayzed-datepicker#13]
I'm getting warnings saying that the days of the month keys, e.g. "72022S", are duplicated, as well as a significant performance hit when I define `configs`.

**Comment by aboveyunhai** [comment]
are u able to replicate the issue in codesandbox https://codesandbox.io/s/?

**Comment by david-morris** [comment]
Something that might be useful --- I've noticed that the duplicated keys issue goes away if I memoize the config, but when I do that, and only change them once, new monthNames aren't being honored.  Also, the performance hit is still there.  This is a problem, since I use react-i18n and want to show customers month names in their native language.

**Comment by david-morris** [comment]
Ok!  I've noticed that you have a key scheme which depends on the string values of days and months, rather than their indexes.
I tracked the performance hit down --- I labeled the days of the week S M T W T F S. When I changed it to S M T W D F Sa it worked again.  I'll see if I can get  PR ready to use index numbers instead of strings, but it will probably end up combined with my existing open PR about the README if that's OK.
